packages/atlas/src/components/_video/VideoHero/VideoHero.styles.ts

```diff
@@ -92,3 +93,7 @@ export const ButtonsContainer = styled.div`
     flex-shrink: 0;
   }
 `
+
+export const TitleText = styled(Text)<IsCategoryProp>`
+  ${({ isCategory }) => (!isCategory ? `overflow-y: auto` : ``)};
```

Comment:
Let's avoid this type of conditional style inside a string. Please refer to https://github.com/Joystream/atlas/blob/master/docs/styleguide.md#conditional-styles

Reply:
Fixed :)